### PR TITLE
Improve script

### LINKED ISSUE #4: wget not using the raw path
The path for wget in the following doc is wrong:
https://github.com/hoprnet/hopr-sh/blob/main/README.md

```JS
wget https://github.com/hoprnet/hopr-sh/blob/master/install-uni.sh
```
Should be:
```JS
wget https://raw.githubusercontent.com/hoprnet/hopr-sh/main/install-uni.sh
```

### PR DESCRIPTION
**Pull request by nionis**
Fixes #4

### CODE REVIEW

File: setup-hoprd-macos.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    read -p "Do you agree to our Privacy Policy? [y/n]:" response
+    case $response in
+        y) echo Terrific!
+        echo -e ${RED}Warning! Running this script repeatedly will cause you to have a new node address each time.${NC}
+
+        read -p "Would you like to run this script? [y/n]:" response
+        case $response in
+            y) echo What release are you installing? Format: X.XX.X \(https://github.com/hoprnet/hoprnet/releases\)
+                read vrelease
+                brew install node@12
```
Comment (jjperezaguinaga) [inline setup-hoprd-macos.sh:26]: @nionis I believe there might be some additional requirements (e.g. `gcc`) that I'm not 100% might be pulled from `brew`. Did we try in a brand new device? Also, I believe @kitschbent already figured out which commands work on `macOS` w/o being dead slow.

**Review by jjperezaguinaga — APPROVED** [review]
LGTM, single comment about improvements for macOS

File: setup-hoprd-macos.sh (hunk 1 above)
Comment (nionis) [reply setup-hoprd-macos.sh:26]: @jjperezaguinaga any recommendations on how to test it on a brand new machine?
the script @kitschbent send me doesn't include all instructions needed to get hoprd running, for example installing nodejs is missing.
I don't have much experience running on macOS, but AFAIK brew installs all necessary deps.

**Comment by kitschbent** [comment]
We did install brew from scratch yes, and then we also brewed 'apt' rather
than use curl. `brew install apt` i think was it, no flags

**Comment by nionis** [comment]
I will merge this for now as it contains many improvements, opening issue #6

**Pull request merged**